### PR TITLE
Balder/handle timeout properly for batched send

### PR DESCRIPTION
**Pull request by baldersheim**
@geirst PR

### CODE REVIEW

**Comment by baldersheim** [comment]
@geirst Now all tests are verified

**Comment by geirst** [comment]
@vekterli FYI

File: vespaclient-core/src/main/java/com/yahoo/feedapi/MessagePropertyProcessor.java [hunk 1]
```diff
@@ -114,6 +116,7 @@ public PropertySetter buildPropertySetter(HttpRequest request) {
             if (timeoutStr != null) {
                 timeoutParam = Double.parseDouble(timeoutStr);
             }
+            useConstantTimeout = request.getBooleanProperty("useconstanttimeout");
```
Comment (geirst) [inline vespaclient-core/src/main/java/com/yahoo/feedapi/MessagePropertyProcessor.java]: I guess this returns false if the property is not found?

**Review by geirst — APPROVED** [review]
👍

**Comment by geirst** [comment]
@vekterli please review as well

**Review by vekterli — APPROVED** [review]
:+1:

**Pull request merged**